packages/repository/src/__tests__/unit/model/model.unit.ts

```diff
@@ -444,6 +444,31 @@ describe('model', () => {
     });
   });
 
+  it('hide a propety with PropertyDefinition', () => {
```

Comment:
`hide a propety with PropertyDefinition` -> `hides a property defined as hidden`

Reply:
> @mdbetancourt please consider this approach. It will be less complex and less disruptive.

yup i already used that approach
> hide a propety with PropertyDefinition -> hides a property defined as hidden

got it